index.html

```diff
@@ -104,53 +105,79 @@ <h1>Plug and play</h1>
-            </p>
-            <p>
-                We work well within a budget, and provide the same value cheaper than any other option.
-            </p>
-            <div class="read-more">Get started right</div>
-        </a>
-        <a href="/start-automating/" class="product product-automation">
-            <img src="/img/input-icon.svg">
-            <h1>Automate away the tiresome tasks</h1>
-            <p>
-                Business is going well, but as your company has grown, more and more time is spent on drudge work.
-                We can help you identify and automate boring work, freeing up more time for your employees to focus on what is important.
-            </p>
-            <p>
-                We help ensure a constructive process for and buy-in from all stakeholders.
-            </p>
-            <div class="read-more">Start automating</div>
-        </a>
-        <h1 class="product-problem-header">Got a problem?</h1>
-        <a href="/get-awesome/" class="product product-fix-platform">
-            <img src="/img/layers-icon.svg">
-            <h1>Make your existing platform awesome!</h1>
-            <p>
-                If you already have a tech product we can help you maintain it, expand it with new features, and prepare it for growth.
-                We ensure that maintenance is efficient and stays within your budget.
+    <h1 class="title">What we excel at</h1>
+    <div class="segment">
+        <div class="icon icon-top arrow-1">☈</div>
+        <div class="problem">
+            <h2 class="mt-0">Setting the right direction</h2>
+            It's hard to find the kind of person who can see the big picture of your business,
+            and make the right technical decisions to support it. Even if you have a great CTO,
+            they will quickly get too many tasks for each day. That leads to high-level technical
+            decision making and direction setting becoming a bottleneck.
+        </div>
+        <section class="customer-quote-section">
+            <p class="quote-text my-0">
+                Working with <a href="mailto:niels@deranged.dk">Niels</a> has been an absolute pleasure! His ability to absorb and assimilate new information into actionable strategy is beyond compare. Even more impressive are his exceptional articulation and communication skills. <!--Choosing to work with Niels could very well be the best decision you  make for your company.-->
             </p>
-            <p>
-                We can guide you through technical tradeoffs, and help you find new ways to increase cash flow for your product.
+            <p class="quotee mb-0">Isaac, Product Owner</p>
+        </section>
+        <div class="solution">
+            <h2 class="mt-0">[our solution]</h2>
+            Our consultants take on that exact perspective, to take on the role as temporary
```

Comment:
Tekstforslag: "Our consultants take on a full perspective of your company and industry, to take on the role as temporary"